decidim-elections/app/types/decidim/elections/election_answer_type.rb

```diff
@@ -17,6 +17,8 @@ module Elections
       field :title, !Decidim::Core::TranslatedFieldType, "The title for this answer"
       field :description, Decidim::Core::TranslatedFieldType, "The description for this answer"
       field :weight, types.Int, "The ordering weight for this answer"
+      field :votes_count, types.Int, "The votes for this answer"
+      field :selected, types.Boolean, "Is this answer selected?"
```

Comment:
It's a good practice to tell if the filed can be `null` or not using `null: false`. It's very useful for schema introspection 😄 (I see the other fields doesn't have the `null` property so we can do this later for all the fields).